python/examples/sample-selection.py

```diff
@@ -0,0 +1,74 @@
+import sys
+
+import chemfiles
+
+from rascaline import SoapPowerSpectrum
+from equistore import Labels, TensorMap
+import numpy as np
+
+with chemfiles.Trajectory(sys.argv[1]) as trajectory:
+    frames = [f for f in trajectory]
+
+HYPER_PARAMETERS = {
+    "cutoff": 5.0,
+    "max_radial": 6,
+    "max_angular": 4,
+    "atomic_gaussian_width": 0.3,
+    "center_atom_weight": 1.0,
+    "radial_basis": {
+        "Gto": {},
+    },
+    "cutoff_function": {
+        "ShiftedCosine": {"width": 0.5},
+    },
+}
+
+calculator = SoapPowerSpectrum(**HYPER_PARAMETERS)
+
+descriptor = calculator.compute(frames)
+
+# The names of the samples that will be chosen during training
+# coincide with the samples of the blocks obtained after calculating the
+# properties. Accordingly, the names of the samples can be obtained with the
+# following commands (usually 'structure' (structure number) and 'center'
+# (atom number in the structure).
+print(f"samples names: {descriptor.block(0).samples.names}")
```

Comment:
```suggestion
print("samples names:", descriptor.samples_names)
```

Reply:
This function doesn't work for me, `samples_names` is not a `TensorMap` attribute. Surely it is already added to the repository? I installed the latest version of equistore, but that doesn't help.